### DISCUSSION TITLE
Github.com and github.io : amazing inconsistence - and a difference in these both views - why on earth !?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#23355] [Other Feature Feedback, Questions, & Ideas]
<p>hello dear friends,</p>
<p><a href="http://github.com" rel="noopener nofollow ugc">github.com</a> and <a href="http://github.io" rel="noopener nofollow ugc">github.io</a> : amazing inconsistence - and a difference in these both views - why on earth !?</p>
<p>the question is: i still wonder why i have such different pages - with a very different content - see the pages</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://fsj-digital.github.io/pages/">
  <header class="source">

      <a href="https://fsj-digital.github.io/pages/" target="_blank" rel="noopener nofollow ugc">pages</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://fsj-digital.github.io/pages/" target="_blank" rel="noopener nofollow ugc">Willkommen auf den GitHub Pages für das FSJ-Digital des Internationalen Bundes</a></h3>

  <p>Demonstration or Explanation of topics for self-testing and distribution purposes</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/4/34f584288798df4e25aea63c6feae4270da5e359.jpeg" data-download-href="/uploads/short-url/7yuQmBc49Q0YZcgVbmcdcxnw3TH.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/32641183/181076493-fb95ce4d-f915-48a5-ba32-9fc232d3a4db.jpeg" alt="image" data-base62-sha1="7yuQmBc49Q0YZcgVbmcdcxnw3TH" width="690" height="487" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/4/34f584288798df4e25aea63c6feae4270da5e359_2_690x487.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/3/4/34f584288798df4e25aea63c6feae4270da5e359_2_1035x730.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/4/34f584288798df4e25aea63c6feae4270da5e359.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/4/34f584288798df4e25aea63c6feae4270da5e359_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1177×831 119 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>and see here: if i view it on github <a href="https://github.com/fsj-digital/pages" class="inline-onebox" rel="noopener nofollow ugc">GitHub - fsj-digital/pages: Demonstration or Explanation of topics for self-testing and distribution purposes</a></p>
<p>it looks totally different - i have no glue why this happens…</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/6/66989d1600db36020d3ed9c642d9d9ff7c9c21e1.png" data-download-href="/uploads/short-url/eDBFRhGclpzHK1eCxwybSlGlcT7.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/32641183/181076494-8a178915-33c7-4dbf-9ff6-3b8d12396f8e.png" alt="image" data-base62-sha1="eDBFRhGclpzHK1eCxwybSlGlcT7" width="641" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66989d1600db36020d3ed9c642d9d9ff7c9c21e1_2_641x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66989d1600db36020d3ed9c642d9d9ff7c9c21e1_2_961x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/6/66989d1600db36020d3ed9c642d9d9ff7c9c21e1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66989d1600db36020d3ed9c642d9d9ff7c9c21e1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1199×935 72.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>any idea why there is such a amazing inconsistence - and a difference in these both views</p>

**Comment by tcbyrd** [comment, marked as answer]
<p>Your GitHub Pages site is being published from the <code>gh-pages</code> branch of your repository: <a href="https://github.com/fsj-digital/pages/tree/gh-pages" class="inline-onebox">GitHub - fsj-digital/pages at gh-pages</a></p>
<p>That branch was likely created by the theme chooser when you enabled Pages, so it’s not being updated when you push content to <code>main</code>. If you want the content in your main branch to be what gets pushed to <code>github.io</code>, you’ll need go to into your Pages settings and change the source branch to <code>main</code>.</p>
<p>The documentation on this feature is here: <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site" class="inline-onebox">Configuring a publishing source for your GitHub Pages site - GitHub Docs</a></p>

**Comment by fsj-digital** [comment]
<p>hello dear TCbyrd,<br>
first of all: many many thanks for the quick reply. I am very glad to hear from you.  Your explanation and description of a solution looks very good and impressive.  and yes convincing.</p>
<p>i guess that i am allmost there:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/2/3234186e3058c84288ca2ba9a9ec34e360519bf9.png" data-download-href="/uploads/short-url/7a7udPB2b2qoHlprq021W7zlF2N.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/32641183/181076499-9142b93a-8476-49eb-9bde-ca7c0445fe44.png" alt="image" data-base62-sha1="7a7udPB2b2qoHlprq021W7zlF2N" width="689" height="449" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/2/3234186e3058c84288ca2ba9a9ec34e360519bf9_2_689x449.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/2/3234186e3058c84288ca2ba9a9ec34e360519bf9_2_1033x673.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/2/3234186e3058c84288ca2ba9a9ec34e360519bf9.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/2/3234186e3058c84288ca2ba9a9ec34e360519bf9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1206×786 49 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>according your description i guess that i have to choose a branche here.</p>

**Comment by fsj-digital** [comment]
<p>hi there - many many thanks.</p>
<p>youre right. You hit the point. It was exactly like you said.</p>
<p>many many thanks for all you did</p>
<p>This is so awesome here.</p>
<p>keep up the great support site.<br>
:wink:</p>

**Comment by tcbyrd** [comment]
<p>No problem at all. Glad I could help!</p>
